src/main/java/cn/nukkit/block/BlockGrassPath.java

```diff
@@ -57,6 +59,9 @@ public int onUpdate(int type) {
     @Override
     public boolean onActivate(Item item, Player player) {
         if (item.isHoe()) {
+            if(player != null){
+                player.getLevel().addSound(player, Sound.USE_GRASS);
```

Comment:
Probably better to play the sound AFTER the action is executed for consistent responses